Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[workspace]
+members = [
+    "cli",
+    "lib"
+]
+
+[workspace.package]
+version = "0.1.0"
+authors = ["Aarav Mehta <aarav05mehta@gmail.com> (Google Summer of Code Participant)", "Pavel Nikonorov <pavel@genxt.network> (Google Summer of Code Mentor; GENXT)", "ELIXIR Cloud & AAI <cloud-service@elixir-europe.org> (ELIXIR Europe)"]
+edition = "2021"
+readme = "./README.md"
+license-file = "LICENSE.txt"
```

Comment:
Isn't this just "LICENSE"? In any case, "LICENSE" without extension is by far the most common filename for licenses in GitHub repos.